source/class/qx/bom/webfonts/WebFont.js

```diff
@@ -93,6 +102,7 @@ qx.Class.define("qx.bom.webfonts.WebFont", {
     _onWebFontChangeStatus : function(ev)
     {
       var result = ev.getData();
+      this.setValid(!!result.valid);
```

Comment:
Are you sure this accomplishes the goal? The `changeValid` event will not be fired if the property is set to the same value it already has. Is there any possibility of that occurring here?